tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -127,6 +127,17 @@ then
   expect_fail pulp rpm repository sync --repository "cli_test_rpm_repository" --sync-policy foobar
 fi
 
+if pulp debug has-plugin --name "rpm" --min-version "3.25.0.dev"
+then
+  expect_succ pulp rpm publication create --repository "cli_test_rpm_repository" --checksum-type sha512
```

Comment:
You added the option for repository. I think the tests are failing, because this is the publication command.